File: src/nodejs/extras/load_env.js
module.exports = function(app){

	app.get('/env', function(req, res){
		const the_variable=process.env.THE_VAR || "default value";
		res.send(`<div style="text-align: center;"> <h1 >The Variable is:<br> ${the_variable} </h1><br><h3>- NodeJS</h3> </div>` );
	});

}

File: src/python/extras/load_env.py
import sys, os
from __main__ import app



@app.route("/env")
def env():
    the_variable=os.getenv("THE_VAR","default value")
    return f"<div style='text-align: center;'> <h1 >The Variable is:<br> {the_variable} </h1><br>- Python </div>"
